example/setting_example.json

```diff
@@ -1,7 +1,8 @@
 {
   "client_id": "<your client id here>",
   "client_secret": "<your client secret here>",
+  "port": 3001,
   "url":  {
-    "redirect": "http://localhost/connect/res"
+    "redirect": "http://localhost:3001/connect/res"
```

Comment:
mmm, again. This file should be as simple as possible. The port issue is related with your particular situation where you are running two or web app so you have to set different ports to can ... etc. But it doesn't imply that everybody will have same issue.
For this reason is that this file is named `setting_example.json` like a guide to create a new `setting.json` file:
https://github.com/Automattic/node-wpcom-oauth/blob/master/example/index.js#L14
https://github.com/Automattic/node-wpcom-oauth/tree/master/example#2---create-a-setting-json-file